doc/newsboat.asciidoc

```diff
@@ -1405,6 +1405,30 @@ Once an application is properly detached, other apps can be launched from
 the same instance of Newsboat. Each time, you can decide to either keep the same
 browser, or change it to serve a different purpose.
 
+==== Handling Seperate Link Types
```

Comment:
Why create a new sub-section when the docs already mention `%u` at the start of the section itself? I'd extend the existing documentation instead of writing a new subsection and leave the previous one incomplete.